examples/rtc_wakeup.rs

```diff
@@ -32,7 +32,9 @@ fn main() -> ! {
     let mut exti = Exti::new(dp.EXTI);
     let mut pwr = PWR::new(dp.PWR, &mut rcc);
 
-    let mut rtc = Rtc::new(dp.RTC, &mut rcc, &mut pwr, None).unwrap();
+    // If your board has an external crystal, you probably want to select
+    // LSClock::LSE instead of LSClock::LSI below for higher accuracy
```

Comment:
I realise the rest of the docs don't follow this tone but for new docs, please use the passive voice:

```suggestion
    // If the target hardware has an external crystal, RtcClockSource::LSE can be used instead of 
    // RtcClockSource::LSI for greater accuracy.
```

(suggestion wrapped at 100 lines. rustfmt may have other ideas)